frontend/src/store/modules/LookupStore.ts

```diff
@@ -23,6 +23,7 @@ const state = {
   infrastruktureinrichtungTyp: [] as LookupEntryDto[],
   artGsNachmittagBetreuung: [] as LookupEntryDto[],
   sobonOrientierungswertJahr: [] as LookupEntryDto[],
+  sobonOrientierungswertJahrWithoutStandortabfrage: [] as LookupEntryDto[],
```

Comment:
Da die meisten Variablen auch eher eine Zusammensetzung deutscher Wörter sind, würde ich das hier ebenfalls so machen.
```suggestion
  sobonOrientierungswertJahrOhneStandortabfrage: [] as LookupEntryDto[],
```

Reply:
In den Coding-Guidelines ist definiert, dass fachliche Benennungen in Deutsch und der Rest in Englisch geschrieben wird.